modules/gdnative/godot/godot_string.h

```diff
@@ -116,6 +116,44 @@ double GDAPI godot_string_to_double(const godot_string *p_self);
 godot_real GDAPI godot_string_to_float(const godot_string *p_self);
 godot_int GDAPI godot_string_to_int(const godot_string *p_self);
 
+double GDAPI godot_string_char_to_double(const char *p_what);
+//static double to_double(const CharType *p_str, const CharType **r_end = NULL);
+godot_int GDAPI godot_string_char_to_int(const char *p_what); //(int p_len = -1)
```

Comment:
I don't know how to call static and non-static method with the same name... any idea ?
Ex:
`to_int()` and `static to_int()`